src/components/case_manager/AllParas.tsx

```diff
@@ -26,30 +27,10 @@ const AllParasPage = () => {
     return <div>Loading...</div>;
   }
 
+  // reusable Functional component for form only
```

Comment:
```suggestion
```

Reply:
Thank you, @codetheweb ! I made the changes and pushed my edits up!